src/app/index.spec.tsx

```diff
@@ -45,6 +45,19 @@ describe('create app', () => {
     expect(app.history.location.state).toEqual('asdf');
   });
 
+  it('adds sentry middleware when enabled', () => {
+    const actualSentry = jest.requireActual('../helpers/Sentry').default;
+    const mockedSentry = require('../helpers/Sentry').default;
+    mockedSentry.shouldCollectErrors = true;
+
+    const initializeWithMiddleware = jest.spyOn(mockedSentry, 'initializeWithMiddleware');
+    initializeWithMiddleware.mockReturnValue(actualSentry.initializeWithMiddleware());
```

Comment:
you should use a mocked middleware here instead of the real one. in this case you may be able to get away with just setting it to `jest.fn()`

Reply:
I tried a bunch of stuff, and just `jest.fn()` definitely doesn't work